plugins/org.eclipse.fordiac.ide.gef/src/org/eclipse/fordiac/ide/gef/utilities/MostRecentlyUsedTracker.java

```diff
@@ -0,0 +1,233 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Wolfgang Schedl.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Wolfgang Schedl - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.fordiac.ide.gef.utilities;
```

Comment:
I would prefer if this calls is moved to `org.eclipse.fordiac.ide.model.typelibrary`. And that an instance of a `TypeLibraryObject`will maintain an instance of this class. This would give you easy access to the project and also the lifetime of the MRU tracker can be better controlled.